### ISSUE TITLE
svg font size units

### ISSUE THREAD

**Issue by miminno** [jfree/jfreesvg#3]
Shouldn't the font size for text elements be in points rather than pixels? getSVGFontStyle() prints 'px' for the 'font-size' attribute.

**Comment by sladkoff** [comment]
I don't think that it should.

It states [here](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/font-size) that the value of `font-size` can actually be any of the following: 
`<absolute-size> | <relative-size> | <length> | <percentage> | inherit`

If anything, the font-size unit could be made configurable, I guess.

**Comment by miminno** [comment]
Yes, exactly, java.awt.Font is in points and the svg font size should be the same. I should've mentioned that in the original post.

**Comment by miminno** [comment]
Bump. Any thoughts on this? Thanks.

**Comment by jfree** [comment]
I think you are correct.  I'll do some investigation and change it.

**Comment by jfree** [comment]
In fact, 'px' is working better that 'pt' at the moment because the other dimensions in the SVG are defaulting to 'px' (so everything is consistent, even it not matching the Java2D standard unit size).  I'll continue analysing this alongside the viewBox request that is also outstanding.

**Comment by miminno** [comment]
For me this causes big issues when exporting the same JFreeChart chart to PDF and SVG. Fonts are different and as a result the whole chart looks different. I'm attaching the sample for comparison here.
I realize that switching from px to pt is not that easy due to all internal calculations that take place, but I hope that perhaps there is some reasonable solution to this.

![svgvspdf](https://cloud.githubusercontent.com/assets/2623465/20012210/b69fc11a-a284-11e6-968e-ed44f9503a71.png)
